src/components/button/button.md.scss

```diff
@@ -484,6 +484,12 @@ $button-md-strong-font-weight:                  bold !default;
   transition-timing-function: ease-in-out;
 
   pointer-events: none;
+
+  @include multi-dir() {
+    top: 0;
+
```

Comment:
Can you remove this empty line + move `//scss-lint:disable...` to the first row of the mixin? something like
```
@include multi-dir() {
  // scss-lint:disable PropertySpelling
  top: 0;
  left: 0;
}
```

(because we don't won't to ignore it for the entire block)